spec/spec_helper.rb

```diff
@@ -17,3 +17,7 @@
 Dir["spec/fake_app/**/*.rb"].each do |file|
   require File.join(File.dirname(__FILE__), "..", file)
 end
+
+RSpec.configure do |config|
```

Comment:
Fixed other issues you laid out previously as well, thanks @icco 

What needs to be done at this point to get this released - @geemus any other feedback?